tests/apollo/test_skvbc_preexecution.py

```diff
@@ -369,7 +367,7 @@ async def test_alternate_f_isolated(self, bft_network, tracker):
 
     async def issue_tracked_ops_to_the_system(self, tracker):
         try:
-            with trio.move_on_after(seconds=30):
-                await tracker.run_concurrent_ops(10, write_weight=1)
+            with trio.move_on_after(seconds=40):
+                await tracker.run_concurrent_ops(100, write_weight=1)
```

Comment:
If you're increasing the view change timeout, can we then reduce the number of concurrent ops to 50? I noticed that with your PR, the tests take ~40min, as opposed to ~35min on master...

Reply:
PS: otherwise, the rest looks good to me!